lighthouse-core/audits/byte-efficiency/unminified-css.js

```diff
@@ -12,10 +12,50 @@ const computeTokenLength = require('../../lib/minification-estimator.js').comput
 
 const UIStrings = {
   /** Imperative title of a Lighthouse audit that tells the user to minify (remove whitespace) the page's CSS code. This is displayed in a list of audit titles that Lighthouse generates. */
-  title: 'Minify CSS',
-  /** Description of a Lighthouse audit that tells the user *why* they should minify (remove whitespace) the page's CSS code. This is displayed after a user expands the section to see more. No character length limits. 'Learn More' becomes link text to additional documentation. */
-  description: 'Minifying CSS files can reduce network payload sizes. ' +
-  '[Learn more](https://developers.google.com/web/tools/lighthouse/audits/minify-css).',
+  title: {
+    message: 'Minify CSS like {css}',
+    placeholders: {
+      css: '`<link rel=stylesheet>`',
+    },
+  },
+  /** (Message Description goes here) Description of a Lighthouse audit that tells the user *why* they should minify (remove whitespace) the page's CSS code. This is displayed after a user expands the section to see more. No character length limits. 'Learn More' becomes link text to additional documentation. */
+  description: {
+    message: 'Minifying CSS files can reduce network payload sizes. {link_start}Learn More!!!{link_end}. This audit took {milliseconds} ms.',
+    placeholders: {
+      link_start: '[->',
+      link_end: '](https://developers.google.com/web/tools/lighthouse/audits/minify-css)',
+      /** 520 (Placeholder examples go here) */
+      milliseconds: '{timeInMs, number, milliseconds}',
+    },
+  },
+  /** [ICU Syntax] Some plural warning... */
+  warningPlural: '{itemCount, plural, ' +
+    '=1 {# error found}' +
+    'other {# errors found}}',
+  /** [ICU Syntax] Some gendered (ICU select) explanation... */
+  explanationGender: {
+    message: 'Someone minified this, {direct_replace_name}. {static_replacement} {person, select, ' +
```

Comment:
This contains a 1. direct replacement in `{ICU string replace}` a `static_replacement` that is covered by a placeholder, and "Google valid" `ICU select` syntax.